nominations/tests.py

```diff
@@ -1,3 +1,87 @@
 from django.test import TestCase
+from models import Application
+from local_groups.models import Group
 
-# Create your tests here.
+
+class ApplicationTestCase(TestCase):
+    def setUp(self):
+        # create group
+        group = Group.objects.create(
+            name='Test Group',
+            group_id=0000
+        )
+
+        # create application
+        Application.objects.create(
+            group=group,
+            rep_email='john@example.com',
+            rep_first_name='John',
+            rep_last_name='Doe',
+            rep_phone='6155555555',
+            candidate_first_name='Candidate',
+            candidate_last_name='Candidateson',
+            candidate_office='Governor',
+            candidate_state='TN'
+        )
+
+    def test_application_creation(self):
+        """
+        When an application is created, a Questionnaire and a Nomination
+        should be created as well.
+        """
+
+        app = Application.objects.first()
```

Comment:
If this is a unit test, then we should test an individual method here? What are we testing, the save method? This is really written more like an integration test than a unit test right now.

Reply:
Yes, these aren't true unit tests. Am rewriting tests now.